docs/getting-started/tooling-installation.md

```diff
@@ -46,7 +46,7 @@ You can install the latest-published version of the Koreo Language Server VS
 Code extension directly from VS Code. Simply open the extensions panel and
 search for “koreo-ls”.
 
-It will assume your language server is at `/usr/local/bin/koreo-ls`. You can
+It will assume your language server is at `koreo-ls`. You can
```

Comment:
Perhaps "It will assume the `koreo-ls` language server executable is in your system path"?